mintapi/api.py

```diff
@@ -173,12 +173,15 @@ def get_email_code(imap_account, imap_password, imap_server, imap_folder, debug=
     'win64': 'win32'
 }
 
+status_message = None
+
 
 def get_web_driver(email, password, headless=False, mfa_method=None,
                    mfa_input_callback=None, wait_for_sync=True,
                    wait_for_sync_timeout=5 * 60,
                    session_path=None, imap_account=None, imap_password=None,
                    imap_server=None, imap_folder="INBOX"):
+    global status_message
```

Comment:
I would love to see this done without the use of globals, even if you have to stick it as a property on the `driver` object that is returned.

Reply:
I changed get_web_driver to return a tuple and I updated the test code to support the changes.